apps/prairielearn/src/ee/pages/instructorAiGenerateDrafts/instructorAiGenerateDrafts.ts

```diff
@@ -0,0 +1,78 @@
+import * as express from 'express';
+import asyncHandler from 'express-async-handler';
+
+import * as error from '@prairielearn/error';
+import { loadSqlEquiv, queryRows } from '@prairielearn/postgres';
+
+import { getCourseFilesClient } from '../../../lib/course-files-api.js';
+import { QuestionSchema } from '../../../lib/db-types.js';
+
+import {
+  InstructorAIGenerateDrafts,
+  draftMetadataWithQidSchema,
+} from './instructorAiGenerateDrafts.html.js';
+
+const router = express.Router();
+const sql = loadSqlEquiv(import.meta.url);
+
+router.get(
+  '/',
+  asyncHandler(async (req, res) => {
+    if (!res.locals.authz_data.has_course_permission_edit) {
+      throw new error.HttpStatusError(403, 'Access denied (must be course editor)');
+    }
+
+    const drafts = (
+      await queryRows(
+        sql.select_draft_generation_info_by_course_id,
+        { course_id: res.locals.course.id },
+        draftMetadataWithQidSchema,
```

Comment:
Conventionally this should start with a capital letter.

Reply:
changed in 15ec6dd

Reply:
Changed in 15ec6dda6dc5f4677ed02a739e8498c6b7fa883a.